mgmt/api/CoreAPI.cc

```diff
@@ -885,10 +885,11 @@ EventSignalCbUnregister(const char *event_name, TSEventSignalFunc func)
  *-------------------------------------------------------------------------
  * Sets the HOST status to Down
  */
+
 TSMgmtError
-HostStatusSetDown(const char *name)
+HostStatusSetDown(const char *req, int len)
```

Comment:
Is "len" here the actual downtime? If so, "len" seems very confusing, looking at this API, I'd assume len here is the length of the const char* req. if it is so, then this is fine, if not, we should change this to down_time or some such.